app/views/comments/_sort_option.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="comment-sort-option">
```

Comment:
> would a `<nav />` with an `aria-label` (to distinguish it from other `<nav />`s) that contains a list of hyperlinks make more sense?

Yes, I think in this case that's most semantic/accessible. Not all lists of links _need_ to be in a `nav`, but I think in this case it could be helpful since they are a very distinct group. So what we want is something like:

```html
<nav aria-labelledby="sort-comments-heading">
  <h3 id="sort-comments-heading">Sort discussion:</h3>
  <ul>
    <li>
      <a href="<link to the sort option>" aria-describedby="<sort option name>-description-text">sort option text</a>
      <span id="<sort option name>-description-text">description text for sort option</span>
    </li>
  <ul>
</nav>
```
This is just pseudo-code, but essentially we want a list of links inside a `nav`, and we want all descriptions to be linked to the link itself with `aria-describedby`. `Top`, `Latest`, etc should not be headings.

Also, `aria-current="page"` should be added to the `<a>` tag when that item is the currently selected sort option (we could even control the appearance of the checkmark icon in CSS with the `[aria-current="page"]` selector)

Let me know if I can provide any more clarification on this!

Reply:
@aitchiss Are you implying that we also load the svg from CSS? 

How would you set color to the svg via css. For example I tried some code 

HTML
```
<li class="comment-sort-option">
  <a href="/admin_mcadmin/this-is-title-2m9f?comments_sort=top" class="comment-sort-option__header first-nav-link" aria-describedby="top-description-text" aria-current="page">
    Top
  </a>
  <span id="top-description-text" class="crayons-field__description">Most upvoted and relevant comments will be first</span>
</li>
```

Css
```
    [aria-current="page"]::before{
      content: '';
      background-image: url(image_path('checkmark.svg'));
      position: relative;
      display: block;
      background-size: 28px 28px;
      height: 28px;
      width: 28px;
    }
```
The problem is that the svg loads but not in the font-color we want.  There's a way I found on [stack](https://stackoverflow.com/a/42317014/3480449).  Should we go that route? 

Other way would be to load the svg in html, but we might end up in code that might be undesirable
```
<li class="comment-sort-option">
  <span class="sort-selected">
      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" class="crayons-icon" role="img" aria-labelledby="abks7e4kr1ixjsoqqloov97nkecs1rbl"><title id="abks7e4kr1ixjsoqqloov97nkecs1rbl">Selected Sort Option</title>
    <path d="M10 15.172l9.192-9.193 1.415 1.414L10 18l-6.364-6.364 1.414-1.414 4.95 4.95z"></path>
</svg>

  </span>
  <span>
    <a href="/admin_mcadmin/this-is-title-2m9f?comments_sort=top" class="comment-sort-option__header first-nav-link" aria-describedby="top-description-text" aria-current="page">
      Top
    </a>
    <span id="top-description-text" class="crayons-field__description">Most upvoted and relevant comments will be first</span>
  </span>
</li>
```

Reply:
Yes sorry I wasn't very clear in my suggestion! We can load the svg in the `html.erb` file, using the `crayons_icon_tag` helper function, adding it inside the link, and including a class name like `sort-selected-icon`, keeping it hidden by default (using the helper class name 'hidden'). We could then reveal it using something like

```scss
[aria-current="page"] {
  .sort-selected {
    display: inline-block;
  }
}
```

Reply:
(but if showing/hiding the icon in this way adds too much complexity, I don't mind going another direction - it was just an idea! 🙂 )

Reply:
changes made.